tic-tac-toe/src/App.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from "react";
+import logo from "./logo.svg";
+import "./App.css";
+import PersistantScore from "./components/PersistantScore";
+import Game from "./components/Game";
+import { Score, Symbol, Winner } from "./utils";
+import EndScreen from "./components/EndScreen";
+
+function App() {
+  let [score, setScore]: [Score, any] = useState({ X: 0, O: 0 });
```

Comment:
1. Use `const`
2. Variable names, lowercase `x` and `o`
3. When typing this, do this: 

```ts
const [score, setScore] = useState<Score>({ x: 0, o: 0});
```